core/src/main/java/org/frankframework/jms/JmsMessageBrowser.java

```diff
@@ -240,13 +240,22 @@ protected String getCombinedSelector(Map<String,String> selectors) {
 		return result.toString();
 	}
 
-
 	/**
 	 * Timeout <i>in milliseconds</i> for receiving a message from the queue
+	 * @deprecated use {@link #setTimeout(long)} instead
 	 * @ff.default 3000
 	 */
+	@Deprecated(since = "8.1")
```

Comment:
Om dit inzichtelijk te maken voor de Frank!Ontwikkelaars kan je dit doen.
```suggestion
	@Deprecated(since = "8.1")
	@ConfigurationWarning("Use attribute timeout instead")
```